src/topp/CometAdapter.cpp

```diff
@@ -209,6 +209,15 @@ class TOPPCometAdapter :
     registerIntOption_("spectrum_batch_size", "<posnum>", 20000, "max. number of spectra to search at a time; use 0 to search the entire scan range in one batch", false, true);
     setMinInt_("spectrum_batch_size", 0);
     registerDoubleList_("mass_offsets", "<doubleoffset1, doubleoffset2,...>", {0.0}, "One or more mass offsets to search (values subtracted from deconvoluted precursor mass). Has to include 0.0 if you want the default mass to be searched.", false, true);
+    registerStringOption_("pinfile_protein_delimiter", "<delimiter>", "\t", "specify a different character or string for the protein column delimiter (default tab)",false, false);
+    
+    for (char residue = 'A'; residue <= 'Z'; ++residue)
+    {
+      std::string set_str = "set_";
+      set_str += residue;
+      set_str += "_residue";
```

Comment:
this code is used in two places. Refactor into a function. Otherwise there is potential for diverging parameter names and nasty crashes.